init.lua

```diff
@@ -2,14 +2,23 @@
 -- Some hardcoded settings and constants
 -----------------------------------------------
 local defaultmodule = "empty"
-local print_to_chat = true
+
+local print_to_chat = minetest.settings:get_bool("qa_block.print_to_chat", true)
+local log_to_file = minetest.settings:get_bool("qa_block.log_to_file", false)
+local overwritelog = minetest.settings:get_bool("qa_block.overwrite_log", false)
+local logdatetime = minetest.settings:get_bool("qa_block.log_date_time", false)
+local datetimeformat = minetest.settings:get("qa_block.date_time_format") or "%Y-%m-%d %H:%M:%S"
```

Comment:
Missing `settingtypes.txt` entries. Documentation and exposure in the Settings tab is important so that its purpose is not forgotten.